book/ch08.md.html

```diff
@@ -64,6 +64,45 @@
 `main()` when we start. When the `color()` routine gets the `hit_record` it can call member
 functions of the material pointer to find out what ray, if any, is scattered.
 
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
+    class sphere: public hitable  {
+        public:
+            sphere() {}
+            sphere(vec3 cen, float r, material *m) : center(cen), radius(r), mat_ptr(m)  {};
```

Comment:
After 86 characters, the Markdeep will render with a scrollbar, so all code samples should be wrapped at 86 characters.

Reply:
By the way, that's why the code fence lines are as long as they are. You can use them as length markers.